### PR TITLE
support country tax rate fallback 

### PR DESCRIPTION
**Pull request by barbara79**
# Description

Closes https://github.com/commercetools/commercetools-jvm-sdk/issues/2034

# Checklist

- [x] Update release Notes
- [ ] Add to the current github milestone 
- [x] Update commercetools api reference

### CODE REVIEW

**Review by katmatt — APPROVED** [review]
Long time no see `countryTaxRateFallbackEnabled` 😄 Looks very good to me 👍

File: commercetools-models/src/test/java/io/sphere/sdk/projects/commands/ProjectUpdateActionsIntegrationTest.java [hunk 1]
```diff
@@ -30,6 +31,21 @@ public void unsetExternalAuthForPerformanceReasons(){
...
+        final ProjectUpdateCommand updateCommand = ProjectUpdateCommand.of(project, ChangeCountryTaxRateFallbackEnabled.of(!countryTaxRateFallbackEnabledActual));
+        final Project updatedProject = client().executeBlocking(updateCommand);
+        assertThat(updatedProject.getCarts().getCountryTaxRateFallbackEnabled()).isEqualTo(!countryTaxRateFallbackEnabledActual);
+        final ProjectUpdateCommand reverseCommand = ProjectUpdateCommand.of(updatedProject, ChangeCountryTaxRateFallbackEnabled.of(countryTaxRateFallbackEnabledActual));
+        final Project reversedProject = client().executeBlocking(reverseCommand);
+        assertThat(reversedProject.getCarts().getCountryTaxRateFallbackEnabled()).isEqualTo(countryTaxRateFallbackEnabledActual);
+
+    }
+
+    @Ignore("Disable because of problems with External OAuth")
```
[the remark this answers is not in the document]
Comment (jenschude) [reply commercetools-models/src/test/java/io/sphere/sdk/projects/commands/ProjectUpdateActionsIntegrationTest.java:48]: It was already disabled by you but on the test class itself :D

File: commercetools-models/src/test/java/io/sphere/sdk/projects/commands/ProjectUpdateActionsIntegrationTest.java [hunk 2]
```diff
@@ -30,6 +31,21 @@ public void unsetExternalAuthForPerformanceReasons(){
         Assertions.assertThat(updatedProject.getExternalOAuth()).isNull();
     }
 
+    @Test
+    public void checkChangeCountryTaxRateFallbackEnabledIsWorking(){
+        final Project project = client().executeBlocking(ProjectGet.of());
+        final Boolean countryTaxRateFallbackEnabledActual = project.getCarts().getCountryTaxRateFallbackEnabled();
+
+        final ProjectUpdateCommand updateCommand = ProjectUpdateCommand.of(project, ChangeCountryTaxRateFallbackEnabled.of(!countryTaxRateFallbackEnabledActual));
```
[the remark this answers is not in the document]
Comment (barbara79) [reply commercetools-models/src/test/java/io/sphere/sdk/projects/commands/ProjectUpdateActionsIntegrationTest.java:39]: thanks :)

File: commercetools-models/src/main/java/io/sphere/sdk/carts/CartsConfiguration.java [hunk 3]
```diff
@@ -6,9 +6,13 @@
...
     
     Integer getDeleteDaysAfterLastModification();
+    Boolean getCountryTaxRateFallbackEnabled();
 
     static CartsConfiguration of(final Integer deleteDaysAfterLastModification) {
         return CartsConfigurationImpl.of(deleteDaysAfterLastModification);
     }
-    
+
+    static CartsConfiguration of(final Boolean countryTaxRateFallbackEnabled) {
```
Comment (jenschude) [inline commercetools-models/src/main/java/io/sphere/sdk/carts/CartsConfiguration.java:15]: Either you should also a `of(final Integer deleteDaysAfterLastModification, final Boolean countryTaxRateFallbackEnabled)` method

or just remove it as there is a specific update action for this flag available and both parameters are optional.

**Review by jenschude — APPROVED** [review]
Small remark

**Pull request merged**